hermes/file_gen.go

```diff
@@ -0,0 +1,38 @@
+package hermes
```

Comment:
Add comment with license header, author, and short description of the purpose of this file.